src/main/java/com/gluonhq/NativeBaseMojo.java

```diff
@@ -68,8 +68,8 @@
     @Parameter(readonly = true, required = true, defaultValue = "${basedir}")
     File basedir;
 
-    @Parameter(property = "client.graalHome")
-    String graalHome;
+    @Parameter(property = "client.graalvmHome")
+    String graalvmHome;
```

Comment:
I know the PR is already merged. My preference would be to rename `graalvmHome` to `graalVMHome`. Since `vm` is a shorthand for Virtual Machine.